src/index.js

```diff
@@ -295,6 +295,7 @@ export const ReactQueryDevtoolsPanel = React.forwardRef(
             style={{
               flex: '1 1 500px',
               minHeight: '40%',
+              maxHeight: '100%',
```

Comment:
Container will extend beyond the dev tools panel without this; this creates a constraint that will generate overflow.